psycopg/psycopg/connection.py

```diff
@@ -851,6 +989,32 @@ def notifies(self) -> Generator[Notify, None, None]:
                 n = Notify(pgn.relname.decode(enc), pgn.extra.decode(enc), pgn.be_pid)
                 yield n
 
+    @contextmanager
+    def pipeline(self) -> Iterator[None]:
```

Comment:
Any reason not to return the Pipeline object? It might be useful... if any for our tests :P

Reply:
Done in 032c43b3a030ca01aa8ff63305d002920aef95b5